docs/resources/dms_rabbitmq_instance.md

```diff
@@ -17,18 +17,47 @@ variable security_group_id {}
 
 data "huaweicloud_availability_zones" "zones" {}
 
-data "huaweicloud_dms_product" "test" {
-  engine        = "rabbitmq"
-  instance_type = "cluster"
-  version       = "3.7.17"
-  node_num      = 3
+data "huaweicloud_dms_rabbitmq_flavors" "test" {
+  type = "cluster"
```

Comment:
```suggestion
data "huaweicloud_dms_rabbitmq_flavors" "test" {
  type              = "cluster"
  storage_spec_code = "dms.physical.storage.ultra.v2"
```

Reply:
It is recommended to specify the disk IO specification, and it is not recommended to use the default.